app/pages/events/view/ActivityView.php

```diff
@@ -1,29 +1,16 @@
 <?php
 restrict_access();
 
-include __DIR__ . "/eventUtils.php";
-
 $event_id = get_route_param("event_id", false);
-$activity = em()->find(Activity::class, get_route_param("activity_id"));
-
-$activity_entry = $activity->entries[0] ?? null;
-$can_edit = check_auth(Access::$ADD_EVENTS);
-$link = $event_id ? "/evenements/$event_id" : "";
+$event = em()->find(Event::class, $event_id);
+if ($event->type == EventType::Simple) {
+    $activity = $event->activities[0];
+} else {
+    $activity = em()->find(Activity::class, get_route_param("activity_id"));
+}
 
-page($activity->name)->css("event_view.css");
 ?>
 
-<?= actions()->back($link ?: "/evenements")
```

Comment:
Have you tried viewing the details of an activity? I feel like this is needed

Reply:
Everything seems to be working fine